test/unit/test_dss_client.py

```diff
@@ -125,6 +125,19 @@ def _files_present(self):
                 for dir_path, _, files in walk('.')
                 for f in files}
 
+    def _mock_download_manifest(self, *args, **kwargs):
+        with patch('hca.dss.DownloadContext._download_file', side_effect=_fake_download_file) as download_func, \
+                patch('hca.dss.DSSClient.get_bundle') as mock_get_bundle:
```

Comment:
I'd break this up into two nested `with` statements. That avoids the weird indention and the backslash.